cmd/encryption.go

```diff
@@ -0,0 +1,186 @@
+package cmd
```

Comment:
Rename this file as just `encryption-v1.go` this just follows the current convention and also indicates that this is the object layer implementation.

Reply:
done